codec/codec_test.go

```diff
@@ -5,18 +5,17 @@
 package codec
 
 import (
+	"github.com/DataDog/temporal-large-payload-codec/server"
```

Comment:
Mind sorting the imports. Usually it is system imports, followed by external dependencies, followed by internal ones.

We might want to setup a linter for that (let's create an issue)